src/os_maild/maild.h

```diff
@@ -47,7 +47,7 @@
                             "Received From: %s\r\n" \
                             "Rule: %d fired (level %d) -> \"%s\"\r\n" \
                             "%s" \
-                            "Portion of the log(s):\r\n\r\n%s\r\n" \
+                            "Alert(s):\r\n\r\n%s\r\n" \
```

Comment:
Let's continue using `Portion of the log(s)`, in many cases is the `full_log` field the printed data.